src/tabs.scss

```diff
@@ -12,16 +12,56 @@
 $block: #{$fd-namespace}-tabs;
 
 .#{$block} {
+  @mixin tabs-semantic($color) {
+    .#{$block}__icon {
+      color: $color;
+      border-color: $color;
+    }
+
+    .#{$block}__link {
+      color: $color;
+
+      @include fd-selected() {
+        color: $color;
+
+        &::after {
+          background-color: $color;
+        }
+
+        .#{$block}__icon {
+          color: var(--sapContent_ContrastIconColor);
+          background-color: $color;
+        }
+      }
+
+      @include fd-hover() {
+        color: $color;
+      }
+
+      &:link {
+        color: $color;
+      }
+    }
+  }
+
   $fd-tabs-margin-bottom: $fd-margin-bottom !default;
   $fd-tabs-border-width: 1px !default;
   $fd-tabs-border-color: fd-color("neutral", 3) !default;
   $fd-tabs-background-color: fd-color("background", 2) !default;
   $fd-tabs-link-color: fd-color("text", 2) !default;
-  $fd-tabs-link-padding-y: fd-space("s") !default;
+  $fd-tabs-selected-color: var(--sapSelectedColor);
+  $fd-tabs-link-padding-y: 0.875rem !default;
+  $fd-tabs-icon-link-padding-y: 1rem !default;
   $fd-tabs-link-padding-x: fd-space("s") !default;
   $fd-tabs-link-border-width: 3px !default;
   $fd-tabs-link-border-color: fd-color("action", 1) !default;
   $fd-tabs-link-transition-params: $fd-animation--speed ease-in !default;
+  $fd-tabs-icon-size: 1.125rem;
+  $fd-tabs-icon-selection-height: 2.625rem;
```

Comment:
should be 2.75rem per speck

Reply:
it's `2.75rem`, but including border, which is `1px` per side